drape_frontend/backend_renderer.cpp

```diff
@@ -605,6 +606,51 @@ void BackendRenderer::AcceptMessage(ref_ptr<Message> message)
       break;
     }
 
+  case Message::Type::Arrow3dRecache:
+    {
+      ref_ptr<Arrow3dRecacheMessage> msg = message;
+      
+      m_arrow3dCustomDecl = msg->GetArrow3dCustomDecl();
+     
+      // Invalidate Arrow3d texture.
+      CHECK(m_context != nullptr, ());
+      m_texMng->InvalidateArrowTexture(m_context,
+                                       m_arrow3dCustomDecl.has_value() ?
+                                         m_arrow3dCustomDecl.value().m_arrowMeshTexturePath :
+                                         std::nullopt,
+                                       m_arrow3dCustomDecl.has_value() ?
+                                         m_arrow3dCustomDecl.value().m_loadFromDefaultResourceFolder :
+                                         false);
```

Comment:
nit: optional has ```value_or``` function

Reply:
In this case, `value_or` should return the struct, propose to use `->` as above